nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 32 Line 3_
```markdown
## .xtra()

The xtra action adds a filter to queries and DDL statements. This makes it easier to limit users (or other objects) access to only things for which they have permission. 
```
Show ddl examples too and explain the details of how they work.

Reply:
I just spotted this comment. 🤦🏻 

For DDL examples, can you explain or provide an example of what you are looking for?

Reply:
Specially, show how `.xtra` changes `insert`, `update`, and `delete`. (Check the fastlite docs/examples and source for details.)

Reply:
Thanks for the coaching! I'm delighted to finally give it a proper go. 😄